### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by almothafar** [tuplejump/play-yeoman#97]
Hello, 

I just wonder is this a dead project? the last update is on 10th of April, and its not working well for play 2.5, and 2.6 almost on way, so is this project a dead project? 

I contacted with @Shiti  and tuplejump from month got no reply until now.

**Comment by malterb** [comment]
good question. fork is always a possibility. Would be good to know what @Shiti intentions are.

**Comment by almothafar** [comment]
@elmalto yes this what I do usually, but sometimes like this plugin, you can't easily fork and modify, I know Java but not Scala for example :) 

If you know good fork for this will be good :) 

I use https://github.com/mmizutani/sbt-play-gulp now, but for grunt projects already I have, it will be good to get updates without spend time to migrate to another plugin, or time to make a fork and fix on my own.

After all, this plugin is nice, and it will be sad story ending if it's simply dead.
